src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int VALUE = 100;
```

Comment:
```suggestion
    private static final int MAX_BALL_NUMBER= 100;
```
names should have sense.